mlfinlab/microstructural_features/third_generation.py

```diff
@@ -0,0 +1,17 @@
+"""
+Third generation models implementation (VPIN)
+"""
+import pandas as pd
+
+
+def get_vpin(volume: pd.Series, buy_volume: pd.Series, window: int = 1) -> pd.Series:
```

Comment:
Is this able to handle data which is very large? Say a 25 gig long csv file.

Reply:
VPIN is calculated on bar series, so there should be no problems with that, unless you use 1s bars for examples